src/iframe/src/Iframe.js

```diff
@@ -478,6 +499,9 @@ class Iframe extends Component {
       // Otherwise, wait for messages from parent.
       window.addEventListener('message', handleMessage)
     }
+
+    this.renderer = params.get('renderer') || "framebuffer"
```

Comment:
I think we should be clearer about what `this.renderer` means. Right now I feel like it's the actual renderer API. We should probably make it `this.useFrameBufferRenderer`.

Reply:
Fixed.